### PR TITLE
chore: fix example app

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:
1. Cleanup s3bucket on cdk destroy
2. Remove snapshot test - the snapshot test is not helping much and is more of a hassle as it requires to update the snapshot whenever dependencies are updated.
3. readme update

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/infrastructure/README.md [hunk 1]
```diff
@@ -60,8 +58,17 @@ rushx cdk:destroy -f
...
 After `rushx cdk:deploy` you can find:
 
 1. USER_POOL_ID [here](./src/config/testEnv.json#L13)
 
-2. DYNAMO_DB_TABLE_NAME [here](./src/config/testEnv.json#L12)
\ No newline at end of file
+2. DYNAMO_DB_TABLE_NAME [here](./src/config/testEnv.json#L12)
+
+
+
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/README.md:74]: if youre in this file again this PR, you can delete these extra lines, but if not dont worry about it

File: workbench-core/example/infrastructure/scripts/cleanup.sh [hunk 2]
```diff
@@ -85,10 +92,33 @@ function cleanup(){
...
+        echo -e "${GREEN}DBName: $DBName${NC}"
+        else
+            echo -e "${RED}*** [ERROR] Could not get the DynamoDB Table***${NC}"
+        fi
+    fi
 }
 
 function sanitize() {
-    echo 'Sanitizing environment variables..'
+    # echo -e "${BLUE}Sanitizing environment variables..${NC}"
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/scripts/cleanup.sh:121]: Was this commented out on purpose?

File: workbench-core/example/infrastructure/scripts/cleanup.sh [hunk 3]
```diff
@@ -32,13 +35,17 @@ Usage: `basename $0`
...
+    -t [GET_DYNAMO_DB_TABLE]
     -c [CLEANUP FLAG]
 
 Example:
-    Delete UserPool:        `basename $0` -u 'us-east-1_abcde' -r 'us-east-1' -c
-    Delete DynamoDB Table:  `basename $0` -d 'TABLE_NAME' -r 'us-east-1' -c
-    Delete Ssm Parameter:   `basename $0` -r 'us-east-1' -p -c
-    Delete All:             `basename $0` -u 'us-east-1_abcde' -d 'TABLE_NAME' -r 'us-east-1' -p -c"
+    Get UserPoolId:                     `basename $0` -r 'us-east-1' -i -c
+    Get DynamoDBTable:                  `basename $0` -r 'us-east-1' -t -c
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/scripts/cleanup.sh:44]: Ah I see now. Im not sure if the cleanup script is the right place to have `get X variable` options

File: workbench-core/example/infrastructure/README.md [hunk 4]
```diff
@@ -60,8 +58,17 @@ rushx cdk:destroy -f
 ```bash
 ./scripts/cleanup.sh -u <USER_POOL_ID> -d <DYNAMO_DB_TABLE_NAME> -r <REGION> -p -c
 ```
+#### Get UserPool Id and DynamoDB Table
+```bash
+./scripts/cleanup.sh -r <REGION> -i -t -c
+```
```
[the remark this answers is not in the document]
Comment (insignias) [reply workbench-core/example/infrastructure/README.md:64]: @bhudnell I used the same script to add more options. Running the above command will not cleanup anything just get the UserPoolId and DynamoDBTable name as you would need this information to cleanup the resources

File: workbench-core/example/infrastructure/scripts/cleanup.sh (hunk 2 above)
Comment (insignias) [reply workbench-core/example/infrastructure/scripts/cleanup.sh:121]: @bhudnell yes i would like to keep it for now. It was just getting overly verbose

**Review by bhudnell — APPROVED** [review]

**Pull request merged**